source/security/private-endpoints.txt

```diff
@@ -214,21 +188,161 @@ connection using either the App Services UI or the Admin API.
+
+   .. tab::
+      :tabid: ui
+
+      .. procedure::
+
+         .. step:: Create a VPC Interface Endpoint
+
+            a. In the App Services UI, navigate to :guilabel:`App Settings` and
+               then click the :guilabel:`Private Endpoints` tab.
+            #. Click :guilabel:`Add Private Endpoint`, then click
+               :guilabel:`Sync`.
+            #. Add your VPC ID and Subnet IDs (comma-separated list) in their
+               separate fields, then click :guilabel:`Next`.
+            #. Copy the generated :guilabel:`VPC Interface Endpoint Command`.
+               The command should look something like this:
+            
+               .. code-block:: sh
+
+                 aws ec2 create-vpc-endpoint \
+                   --vpc-endpoint-type Interface
+                   --service-name <App Services Endpoint Service Name> \
+                   --vpc-id <your-vpc-id> \
+                   --region <your-aws-deployment-region> \
+                   --subnet-ids <your-subnet-ids>
+            #. In your terminal, create a Sync VPC Endpoint with the generated
+               command.
+            #. You must also create a General VPC Endpoint using the command
+               App Services generates for the General Endpoint.
+         
+         .. step:: Add the Private Endpoints to Your App
+
+            Now that you've created the interface endpoint in your VPC, you
+            need to add it to your App. 
+
+            a. Find the Sync and General **VPC Endpoint IDs**, either in the
+               output of the CLI commands you ran in the previous step or on the
+               endpoint details screen of the AWS VPC dashboard. This value
+               starts with ``vpce-``.
+            #. Copy your new VPC Endpoint IDs and paste them into their
+               respective ``Your VPC Endpoint ID`` field in the App Services UI,
+               then click :guilabel:`Next`.
+            #. If this is the first Private Endpoint you are creating, you will 
+               see a toggle you can use to either restrict or open all access from 
+               requests outside your PrivateLink connection. By default, App 
+               Services blocks all requests coming from outside your PrivateLink 
+               connection. This setting applies to this endpoint and **all 
+               additional endpoints you create**. Refer to :ref:`change access 
+               restrictions <change_access_restrictions>` for more information.
+            #. Click :guilabel:`Finish Endpoint Creation`.
+
+   .. tab::
+      :tabid: api
+
+      .. procedure::
+
+         .. step:: Create a VPC Interface Endpoint
+
+            a. Call the :admin-api-endpoint:`List VPC Private Endpoint Services
+               <operation/adminListPrivateEndpointServiceInfo>` endpoint.
+            #. Filter the response object to find the entry for your App's
+               deployment region and the endpoint type you want to create.
```

Comment:
Same Q as above - should we link out to the Cloud Deployment Regions docs here to make it easier for users to figure out the appropriate region, and/or list that endpoint types can be one of `general` and `sync`?